class PasswordChecker {
    private final String password;
    PasswordChecker(String password){
        this.password=password;
    }
    public String getStrength(){
        int len=password.length();
        if(len<6) return "Weak";
        if(len<10) return "Medium";
        return "Strong";
    }
}
